index.html

```diff
@@ -0,0 +1,222 @@
+<!DOCTYPE html>
+<html lang="pl">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <script src="https://kit.fontawesome.com/398c36776a.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header class="header">
+        <div class="header_hero_bg">
```

Comment:
Jeśli ma to być nazwa zgodna z BEM to `header__hero-bg` i ile `header` to `Block`, a `hero-bg` Elemenet